components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabViewHolder.kt

```diff
@@ -39,6 +40,15 @@ abstract class TabViewHolder(view: View) : RecyclerView.ViewHolder(view) {
         styling: TabsTrayStyling,
         observable: Observable<TabsTray.Observer>
     )
+
+    /**
+     * Ask for a partial update of the current tab.
+     * Allows for overriding the current behavior and add or remove the 'selected tab' UI decorator.
+     */
+    open fun updateSelectedTabIndicator(showAsSelected: Boolean) {
+        // no-op.
+        // Not an abstract fun since not all clients of this library might be interested in this functionality.
```

Comment:
nit: Maybe throw an `UnsupportedOperationException` for the default behaviour so a consumer isn't confused if this is called and nothing happens.

Reply:
I like this. Thanks!